databases/migrations/chat-plugins/v5.sql

```diff
@@ -0,0 +1,20 @@
+-- Creates stats tables for the Perspective moderation tool.
+BEGIN TRANSACTION;
+
+CREATE TABLE perspective_stats (
+	staff TEXT NOT NULL,
+	roomid TEXT NOT NULL PRIMARY KEY,
+	result TINYINT(1) NOT NULL,
+	date TEXT NOT NULL, -- YYYY-MM-DD - for easy stat searching.
+	timestamp INTEGER NOT NULL
+);
+
+CREATE INDEX staff_idx ON perspective_stats(staff);
+CREATE INDEX result_idx ON perspective_stats(result);
```

Comment:
What are these two indices for?
You only seem to filter by date and room ID in your code.

Reply:
Futureproofing. Had plans for it - still do - just haven't put them into practice quite yet.

Reply:
I would prefer to not add indices until we have code using them.

Reply:
Ok.

Reply:
Done.